src/main/java/org/opentripplanner/standalone/config/buildconfig/DemConfig.java

```diff
@@ -48,11 +66,29 @@ private static DemExtractParameters mapDemExtract(NodeAdapter config) {
             Unit conversion multiplier for elevation values. No conversion needed if the elevation
             values are defined in meters in the source data. If, for example, decimetres are used
             in the source data, this should be set to 0.1. This overrides the value specified in
-            [`elevationUnitMultiplier`](#elevationUnitMultiplier) in the build config at root level.
+            `demDefaults`.
           """
           )
-          .asDoubleOptional()
-          .orElse(null)
+          .asDouble(defaults.elevationUnitMultiplier())
+      )
+      .build();
+  }
+
+  private static DemExtractParameters mapDefaultDemParameters(NodeAdapter defaults) {
```

Comment:
Same comment as with the `osmDefaults`

Reply:
https://github.com/opentripplanner/OpenTripPlanner/pull/4637/commits/847c55cb7edd22e9fa3493ea71663764ebd288c9